### DISCUSSION TITLE
GD Library extension not available with this PHP installation

### DISCUSSION THREAD

**Discussion by maxfahl** [statamic/cms#5601] [Q&A]
I get the error `GD Library extension not available with this PHP installation` in `statamic/vendor/intervention/image/src/Intervention/Image/Gd/Driver.php`. I can't figure out why this occurs, I have the GD extension enabled in php 7.2.

<img width="517" alt="image" src="https://user-images.githubusercontent.com/19852554/159517689-a0ce3a39-04aa-45be-a48f-05aedc19ffd1.png">

Can anyone give me any pointers here?

**Comment by duncanmcclean** [comment]
To check: if you go into the 'Control Panel -> Utilities -> PHP Info' and search for GD. It's possible you have different installs of PHP running in the terminal than with your web server.
